qa-tests-backend/src/test/groovy/IntegrationsTest.groovy

```diff
@@ -37,10 +34,9 @@ import util.MailServer
 import util.SplunkUtil
 
 import org.junit.Assume
-import org.junit.AssumptionViolatedException
```

Comment:
was this unused?

Reply:
It is in a commented out test. I should have mentioned in the PR description, I ran the IntelliJ 'autofix imports'. Which after the global removal of groups.* seemed like the easiest way to 'clean up' the imports but it also created a lot of shuffling which adds noise to an already noisy PR :(
We could do with a one time application of an opinionated style to these Groovies IMO but I will hold off :)